cargo-apk/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+- The `path` component of the profile signing is now optional, if not specified the `CARGO_APK_<PROFILE>_KEYSTORE` environment variable is read to determine the path to the keystore. ([#358](https://github.com/rust-windowing/android-ndk-rs/pull/358))
```

Comment:
Still a bit confused by the `path` being made optional, but the keystore password not?

Reply:
I guess I'll another environment variable.

Reply:
Thanks, I'd just make the path not optional again so that the user has to specify both or none in `Cargo.toml`.